series_solution.ipynb

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
<p>The number for double is clearly wrong (as you write yourself!) So it is not clear to me how you calculate n here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>